release-notes.md

```diff
@@ -46,6 +80,17 @@ This release has the following security issues:
 
 ### <a id='1-1-resolved-issues'></a> Resolved issues
 
+#### Tanzu CLI - Apps plugin
+* Apps plugin no longer crashes when `KUBECONFIG` includes the colon (`:`) config file delimiter.
+* `tanzu apps workload get`: Passing in `--output json` and `--export` flags together exports the workload in JSON
+rather than YAML.
+* `tanzu apps workload tail`: Duplicate log entries created for init containers have been removed.
+* `tanzu apps workload create/update/apply`
+  * When the `--wait` flag passed and the prompt "Do you want to create this workload?" 
+  is declined, the command immediately exits 0 rather than hanging (continuing to "wait").
+  * Workload name is now validated when the workload values are passed in via `--file workload.yaml`.
+  * When creating/applying a workload from –local-path, if user answers “No” to the prompt “Are you sure you want to publish your local code to [registry name] where others may be able to access it?”, the command now exits 0 immediately rather than showing the workload diff and prompting to continue with workload creation.
```

Comment:
nit: spacing
```suggestion
* `tanzu apps workload create/update/apply`
* When the `--wait` flag passed and the prompt "Do you want to create this workload?" 
  is declined, the command immediately exits 0 rather than hanging (continuing to "wait").
* Workload name is now validated when the workload values are passed in via `--file workload.yaml`.
* When creating/applying a workload from –local-path, if user answers “No” to the prompt “Are you sure you want to publish your local code to [registry name] where others may be able to access it?”, the command now exits 0 immediately rather than showing the workload diff and prompting to continue with workload creation.
```

Reply:
The bullets were intentionally spaced that way because all the items under `tanzu apps workload create/update/apply` refer to updates made for those commands.